### PR TITLE
FAT-12223-C423473

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://folio-org.atlassian.net/browse/FAT-12223
***
https://foliotest.testrail.io/index.php?/cases/view/423473
***
Screenshot_1
***

### CODE REVIEW

File: cypress/e2e/data-import/importing-marc-bib-files/action-profile-is-only-unlinked-once-when-used-in-multiple-times-in-job-profile.cy.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        MatchProfiles.createMatchProfile(matchProfile);
+        MatchProfiles.checkMatchProfilePresented(matchProfile.profileName);
+
+        cy.visit(SettingsMenu.jobProfilePath);
+        JobProfiles.createJobProfile(jobProfile);
+        NewJobProfile.linkMatchProfile(matchProfile.profileName);
+        NewJobProfile.linkActionProfileForMatches(actionProfile.name);
+        NewJobProfile.linkActionProfileForNonMatches(actionProfile.name);
+        // wait for the action profile to be linked
+        cy.wait(1000);
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/importing-marc-bib-files/action-profile-is-only-unlinked-once-when-used-in-multiple-times-in-job-profile.cy.js:97]: Move this waiter to the method

File: cypress/support/fragments/data_import/job_profiles/jobProfileView.js [hunk 2]
```diff
@@ -149,7 +149,41 @@ export default {
       });
   },
 
-  verifyLinkedProfilesNonMatches(arrayOfProfileNames, numberOfProfiles) {
+  verifyLinkedProfilesForMatches(arrayOfProfileNames, numberOfProfiles) {
+    waitLoading();
+    if (numberOfProfiles === 0) {
+      cy.get('[id*="branch-ROOT-MATCH"]').should('not.exist');
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/job_profiles/jobProfileView.js:155]: move to const

**Review by IhorBohdan — APPROVED** [review]

**Review by nayimovag — APPROVED** [review]

File: cypress/e2e/data-import/importing-marc-bib-files/action-profile-is-only-unlinked-once-when-used-in-multiple-times-in-job-profile.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/action-profile-is-only-unlinked-once-when-used-in-multiple-times-in-job-profile.cy.js:97]: I deleted it

File: cypress/support/fragments/data_import/job_profiles/jobProfileView.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/job_profiles/jobProfileView.js:155]: Done

**Pull request merged**